pkg/scheduler/framework/plugins/noderesources/test_util.go

```diff
@@ -22,6 +22,83 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type containerResource struct {
+	cpu    int
+	memory int
+}
```

Comment:
I think the existing package `pkg/scheduler/testing/` should be able to cover all the requirements here, so don't need to introduce a new struct here. See the comments below: